Unit tests fail due to surefire plugin issue with Debian/Ubuntu Java8

See https://issues.apache.org/jira/browse/SUREFIRE-1588

The workaround is to update `maven-surefire-plugin` to version 3.0.0-M1.
```
$ mvn clean install
...
[INFO] --- maven-surefire-plugin:2.22.0:test (default-test) @ biojava-core ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for biojava 5.1.2-SNAPSHOT:
[INFO] 
[INFO] biojava ............................................ SUCCESS [  0.296 s]
[INFO] biojava-core ....................................... FAILURE [  4.512 s]
[INFO] biojava-alignment .................................. SKIPPED
[INFO] biojava-structure .................................. SKIPPED
[INFO] biojava-structure-gui .............................. SKIPPED
[INFO] biojava-genome ..................................... SKIPPED
[INFO] biojava-modfinder .................................. SKIPPED
[INFO] biojava-ws ......................................... SKIPPED
[INFO] biojava-protein-disorder ........................... SKIPPED
[INFO] biojava-aa-prop .................................... SKIPPED
[INFO] biojava-survival ................................... SKIPPED
[INFO] biojava-ontology ................................... SKIPPED
[INFO] biojava-integrationtest ............................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.467 s
[INFO] Finished at: 2018-11-27T15:43:04-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.0:test (default-test) on project biojava-core: There are test failures.
[ERROR] 
[ERROR] Please refer to /working/biojava/biojava-core/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] ExecutionException The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /working/biojava/biojava-core && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -Xmx1500M -jar /working/biojava/biojava-core/target/surefire/surefirebooter686985483942722344.jar /working/biojava/biojava-core/target/surefire 2018-11-27T15-43-02_513-jvmRun1 surefire2664357915002657397tmp surefire_04309217267636059274tmp
[ERROR] Error occurred in starting fork, check output in log
```

--- a/pom.xml
+++ b/pom.xml
@@ -313,7 +313,7 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-surefire-plugin</artifactId>
-					<version>2.22.0</version>
+					<version>3.0.0-M1</version>
 				</plugin>
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>

